README.md

```diff
@@ -94,7 +104,14 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
-
+Calling a function runs it immediately whereas referencing a function waits for an action to set it off (ex. callback).
+function sayHello() {
+  console.log("Hello, Earth.")
+}
+
+Call function: sayHello()
+Reference function:
+  document.body.querySelectorAll("div").addEventListener("click", sayHello)
```

Comment:
:+1: You're not wrong, but it'd be more accurate to say that referencing a function is literally just saying "that function over there", just like referencing a variable.